code/modules/clothing/suits/jobs.dm

```diff
@@ -220,3 +220,34 @@
 	body_parts_covered = ARMS|LEGS|FULL_TORSO|IGNORE_INV
 	allowed = list(/obj/item/weapon/tank/emergency_oxygen,/obj/item/weapon/tank/emergency_nitrogen, /obj/item/device/flashlight,/obj/item/weapon/gun/energy,/obj/item/weapon/gun/projectile,/obj/item/ammo_storage,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/handcuffs,/obj/item/weapon/storage/fancy/cigarettes,/obj/item/weapon/lighter,/obj/item/device/detective_scanner,/obj/item/device/taperecorder)
 	armor = list(melee = 50, bullet = 10, laser = 25, energy = 10, bomb = 0, bio = 0, rad = 0)
+
+//Syndicate exec
+/obj/item/clothing/suit/storage/syndicateexec
+	name = "syndicate executive jacket"
+	desc = "A flash black jacket, it seems oddly heavy."
+	icon_state = "ia_jacket_open"
+	item_state = "ia_jacket"
+	blood_overlay_type = "coat"
+	body_parts_covered = ARMS|LEGS|FULL_TORSO|IGNORE_INV
+	allowed = list(/obj/item/weapon/tank/emergency_oxygen,/obj/item/weapon/tank/emergency_nitrogen, /obj/item/device/flashlight,/obj/item/weapon/gun/energy,/obj/item/weapon/gun/projectile,/obj/item/ammo_storage,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/handcuffs,/obj/item/weapon/storage/fancy/cigarettes,/obj/item/weapon/lighter,/obj/item/device/detective_scanner,/obj/item/device/taperecorder)
+	armor = list(melee = 50, bullet = 10, laser = 25, energy = 10, bomb = 0, bio = 0, rad = 0)
+
+	verb/toggle()
```

Comment:
relative path?

Reply:
Please bare in mind I am an idiot who can barely copy paste code, whats a relative path in this context and how do I fix it?

Reply:
Basically replace that line (which is currently relative pathed) with /obj/item/clothing/suit/storage/syndicateexec/verb/toggle() to make it absolute pathed , and fix the indentation below. Relative pathing requires needs 2 tabs there, absolute just 1

Reply:
Righto, I'll get on that now

Reply:
nope. buggered it up and broke it again